packages/terra-navigation-side-menu/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Default --terra-navigation-side-menu-item-font-weight to normal.
```

Comment:
Can we update this to be under `### Fixed` heading and add a blrub to indicate this chagne was needed to fix a bug?